de.metas.adempiere.adempiere/base/src/main/java-legacy/de/metas/impexp/excel/ArrayExcelExporter.java

```diff
@@ -35,20 +35,23 @@
 	private final Properties m_ctx;
 	private final List<List<Object>> m_data;
 	private final List<String> m_columnHeaders;
+	private final boolean translateHeaders;
 
 	@Builder
 	private ArrayExcelExporter(
 			@Nullable final ExcelFormat excelFormat,
 			@Nullable final ExcelExportConstants constants,
 			@Nullable final Properties ctx,
 			@NonNull final List<List<Object>> data,
-			@Nullable final List<String> columnHeaders)
+			@Nullable final List<String> columnHeaders,
+			@Nullable final Boolean translateHeaders)
 	{
 		super(excelFormat, constants);
 
 		m_ctx = ctx != null ? ctx : Env.getCtx();
 		m_data = data;
 		m_columnHeaders = columnHeaders;
+		this.translateHeaders = translateHeaders != null ? translateHeaders : true;
```

Comment:
```suggestion
		this.translateHeaders = Check.coalesce(translateHeaders, true);
```

Reply:
note: feel free to ignore..it might be just a matter of taste..anyways, i find it easier with coalesce...

Reply:
Done.  Note that coalesce does not come from Check but from org.compiere.util.Util: